spring-cloud-gcp-secretmanager/src/test/java/org/springframework/cloud/gcp/secretmanager/it/SecretManagerPropertySourceIntegrationTests.java

```diff
@@ -49,9 +47,9 @@
 import static org.assertj.core.api.Assumptions.assumeThat;
 import static org.awaitility.Awaitility.await;
 
-public class SecretManagerIntegrationTests {
+public class SecretManagerPropertySourceIntegrationTests {
```

Comment:
Would it be possible to add the `@Value` test here as well like what we're doing in the other PR:

```
	@Test
	public void test() {
		try (ConfigurableApplicationContext c = applicationBuilder.run()) {
			String secret = c.getBean("secret", String.class);
			assertThat(secret).isEqualTo("hello");
		}
	}

	@Configuration
	static class TestConfiguration {

		@Value("${sm://spring-cloud-gcp-it-secret")
		private String secret;

		@Bean
		public String secret() {
			return secret;
		}
	}
```

Reply:
Done.